packages/theming/android-theme/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui-react-native/android-theme",
-  "version": "0.2.7",
+  "version": "0.2.8",
```

Comment:
What you actually want is a changefile, not a manual bump. You can do this by making an edit, running `yarn change` and then reverting your edit. 

But also we're unblocked for now, so let's save this

Reply:
Although maybe just manually bumping the # here is the root cause and we don't want a change file. 🤔.
Maybe this # was always one below what it needed to be?